### PR TITLE
Javalab: add refresh button to code review

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Adds a refresh button to the code review tab. This refreshes all data in the tab that is initially loaded when switching over to the tab.

Screen Shot 2021-10-22 at 4 05 35 PM

## Links

JIRA: https://codedotorg.atlassian.net/browse/CSA-962

## Testing story

Tested locally - checked that new comments and/or updates to comment state appear after clicking refresh.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/i18n/javalab/en_us.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
   "noOtherReviews": "No other projects are ready for review yet.",
   "onlyVisibleToYou": "teacher comments are only visible to you",
   "programCompleted": "Program completed.",
+  "refresh": "Refresh",
```
Comment (jmkulwik) [inline apps/i18n/javalab/en_us.json:81]: Does this already exist in the /common/en_us.json file? If so, we should use the string from there so we can get the already-translated value.

File: apps/test/unit/templates/instructions/ReviewTabTest.jsx [hunk 2]
```diff
@@ -277,6 +278,17 @@ describe('Code Review Tab', () => {
...
         expect(wrapper.find(ReviewNavigator).length).to.equal(1);
       });
+
+      it('reloads data on clicking refresh button', () => {
+        expect(wrapper.state().loadingReviewData).to.be.false;
+        expect(wrapper.find(Button).length).to.equal(1);
+        wrapper
+          .find(Button)
+          .first()
+          .simulate('click');
```
Comment (jmkulwik) [inline apps/test/unit/templates/instructions/ReviewTabTest.jsx]: The enzyme maintainers generally recommend against using simulate on shallow-wrapped components (and often on mounted components too). I believe it's been deprecated in later versions. It's better to invoke the function prop directly instead: https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/invoke.html

File: apps/i18n/javalab/en_us.json (hunk 1 above)
Comment (sanchitmalhotra126) [reply apps/i18n/javalab/en_us.json:81]: Hmm doesn't look like it. Should it be there instead? (it does feel like a fairly common string value)

File: apps/test/unit/templates/instructions/ReviewTabTest.jsx (hunk 2 above)
Comment (sanchitmalhotra126) [reply apps/test/unit/templates/instructions/ReviewTabTest.jsx]: Ah gotcha, will change.

File: apps/i18n/javalab/en_us.json (hunk 1 above)
Comment (jmkulwik) [reply apps/i18n/javalab/en_us.json:81]: Hmm if it's not already there, I generally don't add it to `common` for app-specific strings. We may want to think about whether we eventually want to move all the code review and backpack strings to `common`. That will depend on how likely we are to use this in other apps in the future.

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**